### PR TITLE
Install and configure tailwind

### PR DESCRIPTION
**Pull request by yanqiuzhang**
## Install and configure tailwind css

  _As a developer
  In order to apply styling to our application
  I would like to install tailwind css for user friendly design_

## Tasks:
_Add gem 'webpacker'
Add gem 'tailwindcss'
Installation of tailwind 
Configure application layout_ 

[Github Link](https://github.com/yanqiuzhang/crafty_laundry/tree/setup_styling)

### CODE REVIEW

**Review by NiciP — APPROVED** [review]

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/* eslint no-console:0 */
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+//
+// To reference this file, add <%= javascript_pack_tag 'application' %> to the appropriate
+// layout file, like app/views/layouts/application.html.erb
+
+console.log('Hello World from Webpacker')
```
Comment (vick3d) [inline app/javascript/packs/application.js]: don't think this line is necessary

**Review by vick3d — CHANGES_REQUESTED** [review]

File: app/javascript/packs/application.js (hunk 1 above)
Comment (NiciP) [reply app/javascript/packs/application.js]: ok removed it along with the comments

File: Gemfile [hunk 2]
```diff
@@ -11,6 +11,8 @@ gem 'uglifier', '>= 1.3.0'
 gem 'turbolinks', '~> 5'
 gem 'jbuilder', '~> 2.5'
 gem 'bootsnap', '>= 1.1.0', require: false
+gem 'webpacker', '~> 3.5'
+gem 'tailwindcss', '~> 0.2.0'
 
 group :development, :test do
   gem 'cucumber-rails', require: false
```
Comment (aleciabenjamin) [inline Gemfile]: fix indentation on 17 and 18

File: config/environments/production.rb [hunk 3]
```diff
@@ -1,4 +1,6 @@
 Rails.application.configure do
+  # Verifies that versions and hashed value of the package contents in the project's package.json
```
Comment (aleciabenjamin) [inline config/environments/production.rb:2]: remove these comments?

Comment (NiciP) [reply config/environments/production.rb:2]: ok done

File: Gemfile (hunk 2 above)
Comment (NiciP) [reply Gemfile]: displays correctly on ours locally, cannot change atm

**Review by faraznaeem — APPROVED** [review]

**Review by vick3d — APPROVED** [review]

**Pull request merged**